maven-central-sync/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo pipefail
+
+LOCAL_REPO_DIR=$1
+
+CMD="mvn org.sonatype.plugins:nexus-staging-maven-plugin:1.6.7:"
```

Comment:
could you rename it to something more readable like MVN_CMD so we know what command we're executing

Reply:
good point :)